dev-docs/source/BuildingOnOSX.rst

```diff
@@ -7,11 +7,95 @@ Building on OS X
 .. contents::
   :local:
 
-The following are instructions to build on various versions of OS X:
 
-##################################
+The minimum supported version of macOS is High Sierra (10.13). 
+
+These instructions are from the assumptions of a blank newly installed version of High Sierra using Homebrew for dependency management.
+
+1. Install Xcode 10.1
+
+1. Install Apple's Command Line tools (required by Homebrew)
+
+.. code-block:: sh
+
+   xcode-select --install
+
+1. Install `Homebrew <http://brew.sh>`_.
+
+.. code-block:: sh
+
+   ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+1. Add the necessary 'taps'. The last 4 are to use qt4.
+
+In order to be able to 'tap' the ``mantidproject/mantid`` 'tap' we need to have a couple of packages installed
+
+.. code-block:: sh
+
+   brew install git
+   brew tap homebrew/science
+   brew tap mantidproject/mantid
+   brew tap caskroom/cask
+   brew tap cartr/qt4
+   brew tap-pin cartr/qt4
+   brew cask install xquartz
+   brew cask install mactex
+
+1. Reset brew formula repo back to before Python 2 removal for required packages
+
+.. code-block:: sh
+
+   cd /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core
+   git reset --hard 8f1d612d9e179c0d91ecd09853854e42281bd313
+   cd -
+
+1. Disable homebrew update and install ``mantid-developer`` formula (this may take a while depending on your network speed)
+
+.. code-block:: sh
+
+   HOMEBREW_NO_AUTO_UPDATE=1 brew install mantid-developer
+
+1. Homebrew can stop early for reasons that are unclear. Repeat the above command until Homebrew states: *Warning: mantidproject/mantid/mantid-developer ?.? is already installed and up-to-date*.
+
+1. Pin libraries that depend on Python 2 and libmxml and NeXus is not compatible with v3:
+
+.. code-block:: sh
+
+   brew pin libmxml
+   brew pin sip
+   brew pin pyqt
+
+1. Unlink ``qt@4`` from ``/usr/local`` to avoid cross talk with Qt5
```

Comment:
Could you fix the numbering of the steps? Also you can probably merge the linking of qt.conf and qt-webkit together. I would also put all the historic instructions into a different page, and here at the end, just the link.

Reply:
Do you mean so the raw text doesn't say `1.` everywhere? The rendered document has the numbering correct or would you prefer it correct in the raw markdown? Good point about the historic instructions and merging the other steps.

Reply:
Ahh, I didn't know the numbering works that way.

Reply:
Just using `1.` makes it easier to add/remove lines as the numbering gets automatically updated.

Reply:
I've made the suggested updates to the dev docs.

Reply:
Actually I am totally wrong here. I was getting `.rst` confused with `.md` :man_facepalming: Sorry for the confusion.